figuren-theater/label-printing
# NEW .org/plugin-review action

With the set goal in mind to become a versatile community plugin, _label-printing_ 

> [...] should at **every time** align with the .org/plugin review standards, at least with the automated part of the review process.
> 
> This is where the [Plugin Check (PCP)](https://wordpress.org/plugins/plugin-check/) WordPress plugin comes into play, because thankfully there is also a [corresponding github action](https://github.com/swissspidy/wp-plugin-check-action).
> ### Description of the Change
> 
> [...] we should introduce this action into our default `push` and `pull_request` workflows. This would also be a nice aspect to promote further, saying "We align with the guidelines not only once (like normal), but with every release".
> 
> ~Closes #~

Closes #46 

> ### How to test the Change
> 
> I hope to see the test running within this PR, to get a basic testing.
> ### Changelog Entry
> 
> > Added - Test against standards for the WordPress.org plugin directory

---

Cloned PR from https://github.com/GatherPress/gatherpress/pull/616

---

### Code Checklist
